### PR TITLE
Improve woocommerce docs

### PR DESCRIPTION
**Pull request by gonzunigad**
Improve woocommerce docs

### CODE REVIEW

File: plugin/prestashop/webpay/README.md [hunk 1]
```diff
@@ -21,18 +21,24 @@
...
-</aside>
-
 <h1 class="toc-ignore">Webpay REST Prestashop</h1>
 <h1 style="display: none;">Webpay REST</h1>
 
 ## Descripción
 
 Este plugin oficial ha sido creado para que puedas integrar Webpay fácilmente en tu comercio, basado en Prestashop.
 
+Estás viendo la nueva documentación REST de este plugin. Si aún estás usando la versión anterior SOAP (las que te solicita certificados y una llave privada), revisa este video para actualizarte.
```
Comment (isabellaahc) [inline plugin/prestashop/webpay/README.md:31]: replace => solicita / solicitan

File: plugin/woocommerce/webpay/README.md [hunk 2]
```diff
@@ -21,36 +21,25 @@
...
 Este plugin oficial ha sido creado para que puedas integrar Webpay fácilmente en tu comercio, basado en WooCommerce.
 
-<div class='url-modal-embed' data-toggle-embedYT="modal" data-src="https://www.youtube.com/embed/IeKb8RreF08" >
+Estás viendo la nueva documentación REST de este plugin. Si aún estás usando la versión anterior SOAP (las que te solicita certificados y una llave privada), revisa este video para actualizarte. 
+
+<div class='url-modal-embed' data-toggle-embedYT="modal" data-src="https://www.youtube.com/embed/9--NHgh07Fw" >
   <div class="container-embed">
     <div class="data-info-url">
-      <b>Cómo instalar el plugin de woocommerce</b>
+      <b>Como pasar de SOAP a REST</b>
```
Comment (isabellaahc) [inline plugin/woocommerce/webpay/README.md:36]: replace => Como / Cómo

**Review by isabellaahc — COMMENTED** [review]
replace => (solicita / solicitan) (como / cómo)

File: plugin/prestashop/webpay/README.md (hunk 1 above)
Comment (gonzunigad) [reply plugin/prestashop/webpay/README.md:31]: Thanks! For future reference, you can suggest a change directly in this comment box: 
image

:)

Comment (gonzunigad) [reply plugin/prestashop/webpay/README.md:31]:
```suggestion
Estás viendo la nueva documentación REST de este plugin. Si aún estás usando la versión anterior SOAP (la que te solicita certificados y una llave privada), revisa este video para actualizarte.
```

File: plugin/woocommerce/webpay/README.md (hunk 2 above)
Comment (gonzunigad) [reply plugin/woocommerce/webpay/README.md:36]:
```suggestion
      <b>Cómo pasar de SOAP a REST</b>
```

File: plugin/woocommerce/webpay/README.md [hunk 3]
```diff
@@ -21,36 +21,25 @@
...
-
 <h1 class="toc-ignore">WooCommerce | Plugin Webpay Plus </h1>
 <h1 style="display: none;">Webpay REST</h1>
 
 ## Descripción
 
 Este plugin oficial ha sido creado para que puedas integrar Webpay fácilmente en tu comercio, basado en WooCommerce.
 
-<div class='url-modal-embed' data-toggle-embedYT="modal" data-src="https://www.youtube.com/embed/IeKb8RreF08" >
+Estás viendo la nueva documentación REST de este plugin. Si aún estás usando la versión anterior SOAP (las que te solicita certificados y una llave privada), revisa este video para actualizarte.
```
Comment (gonzunigad) [inline plugin/woocommerce/webpay/README.md:31]:
```suggestion
Estás viendo la nueva documentación REST de este plugin. Si aún estás usando la versión anterior SOAP (la que te solicita certificados y una llave privada), revisa este video para actualizarte. 
```

**Comment by gonzunigad** [comment]
Done @isabellaahc

**Review by afiebig — APPROVED** [review]

**Pull request merged**